index.html

```diff
@@ -319,6 +318,27 @@ <h2 class="about-title">About<br />Myself<br /></h2>
         <ion-icon class="chevron-icons" name="chevron-forward-sharp"></ion-icon>
       </div>
     </section>
+    <section class="contact" id="contact">
+      <div class="inner-div">
+        <h2>Contact me</h2>
+        <p id="inner-div">If you have an application you are interested in developing, a feature that you need
+          built or a project that needs coding. I'd love to help with it </p>
+      </div>
+      <form action="https://formspree.io/f/mvoljlod" id="form" class="form" method="post">
+        <ul class="inner-list">
+          <li><input type="text" class="form-control" placeholder="Full name" name="name" id="name" size="40" maxlength="30">
```

Comment:
Each input field should have the attribute `required`. I think you missed it in the project requirements. Consider adding it to all the input fields.